clipper_admin/clipper_admin/kubernetes/kubernetes_container_manager.py

```diff
@@ -163,6 +168,8 @@ def __init__(self,
         configuration.assert_hostname = False
         self._k8s_v1 = client.CoreV1Api()
         self._k8s_beta = client.ExtensionsV1beta1Api()
+        self._k8s_rbac=client.RbacAuthorizationV1beta1Api()
```

Comment:
Nothing important, but we need spaces for this line.

Reply:
Thanks! I'll make a lint PR formatting the entire code base.